istio.VERSION

```diff
@@ -1,14 +1,14 @@
 # DO NOT EDIT THIS FILE MANUALLY instead use
 # install/updateVersion.sh (see install/README.md)
 export CA_HUB="gcr.io/istio-testing"
-export CA_TAG="87901f278dd40e864b083409dfee9ca72e6a27e9"
+export CA_TAG="4ff5b43394ac9448b564b9b58ae9750a628b933b"
```

Comment:
@sebastienvas this sha errors out

Reply:
this is a proxy sha (!)
https://github.com/istio/proxy/commit/4ff5b43394ac9448b564b9b58ae9750a628b933b